docs/style-guide.rst

```diff
@@ -1058,14 +1058,40 @@ Inside each contract, library or interface, use the following order:
 1. Type declarations
 2. State variables
 3. Events
-4. Modifiers
-5. Functions
+4. Errors
+5. Modifiers
+6. Functions
 
 .. note::
 
     It might be clearer to declare types close to their use in events or state
     variables.
 
+Yes:
+
+.. code-block:: solidity
```

Comment:
So, while this is a good start, as you've put examples into the 'Order of Layout` section, these feel somewhat lacking, as they only show the order of functions and errors, whereas the section describes everything from type declarations, state variables, functions, events, etc.
On the other hand, I'm also not sure how useful the `No` section would be here, as you'd have quite a few permutations of the wrong way to do it, so I think that just having the `Yes` sections (without the actual Yes label) would make more sense. As in:
```
pragma version;

contract C {
  type declarations;
  state variables;
  events;
  errors:
  modifiers;
  functions;
}
```